### PR TITLE
Placeholder for PGA (Hold + Release) Documents

### PR DESCRIPTION
**Pull request by BenjaminMoe**

### CODE REVIEW

**Comment by nissimsan** [comment]
This is good, @BenjaminMoe  - could you add a link to the hold codes somehow?

File: docs/openapi/components/schemas/common/PGARelease.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          - PGARelease
+  reasonCode:
+    type: string
+  description:
+    type: string
+additionalProperties: true
+example: |-
+  {
+    "type": "PGARelease",
+    "reasonCode" : "07",
```
Comment (nissimsan) [inline docs/openapi/components/schemas/common/PGARelease.yml:25]: @BenjaminMoe , are you sure that a code is needed/provided for a release?

**Comment by BenjaminMoe** [comment]
The URL for the reason codes is here: https://www.cbp.gov/sites/default/files/assets/documents/2021-Dec/ACE%20CATAIR%20PGA%20Status%20Notification%20Codes_13Dec2021_508c.pdf

File: docs/openapi/components/schemas/common/PGARelease.yml (hunk 1 above)
Comment (BenjaminMoe) [reply docs/openapi/components/schemas/common/PGARelease.yml:25]: Good point. I'll remove the reason and description on the release and update the PR from draft to review.

File: docs/openapi/components/schemas/common/PGARelease.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+$linkedData:
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/common/PGARelease.yml:1]: Same question as above

**Comment by mkhraisha** [comment]
Can you add the url for the codes you linked above to the description?

**Review by mprorock — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Review by nissimsan — APPROVED** [review]

**Review by mkhraisha — CHANGES_REQUESTED** [review]
Add links to the description of the status pointing to this  https://www.cbp.gov/sites/default/files/assets/documents/2021-Dec/ACE%20CATAIR%20PGA%20Status%20Notification%20Codes_13Dec2021_508c.pdf

**Comment by BenjaminMoe** [comment]
After looking into the requirements more, it looks like hold and release are not separate certificates, but two possible statuses on the same message object.Closing this PR in favor of: https://github.com/w3c-ccg/traceability-vocab/pull/339. The change request from @mkhraisha should be addressed, with adding the document into the description.

**Pull request closed without merging**